docs/compute-systems/casper/remote-desktops.md

```diff
@@ -103,8 +103,12 @@ vncmgr create \
        --job-opts="-l select=1:ncpus=4:mem=20GB"
```

Comment:
Making it easy to add GPU types in the future and providing an example of the syntax is something I'd suggest:

```
!!! example "Creating a VNC session using vncmgr create" 
    === "GP100"
        
        ```
        vncmgr create \
               --account PROJECT \
               --time 2:00:00 \
               --job-opts="-l select=1:ncpus=4:mem=20GB"
        ```

    === "L40"

        Request using `--gpu-model`:
        ```
        vncmgr create \
               --account PROJECT \
               --time 2:00:00 \
               --job-opts="-l select=1:ncpus=4:mem=20GB" \
               --gpu_model=l40
        ```

        Request using `--l gpu_type=l40` in `--job-opts`:
        ```
        vncmgr create \
               --account PROJECT \
               --time 2:00:00 \
               --job-opts="-l select=1:ncpus=4:mem=20GB -l gpu_type=l40" \
        ```
```